UrlManager.php

```diff
@@ -625,7 +625,7 @@ protected function redirectToLanguage($language)
             return;
         }
         Yii::trace("Redirecting to $url.", __METHOD__);
-        Yii::$app->getResponse()->redirect($url);
+        Yii::$app->getResponse()->redirect($url, 301);
```

Comment:
Even though minor, this would still be a BC break. How about making this configurable? E.g.

```php
public $languageRedirectCode = 302;
```

Reply:
> Even though minor, this would still be a BC break. How about making this configurable? E.g.
> 
> ```
> public $languageRedirectCode = 302;
> ```

Good idea. Ready to review.